docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,147 @@
+# Tutorial: Deploy the Chrony charm
+
+## What you'll do
+This tutorial will walk you through deploying the chrony charm; you will:
+1. Deploy and configure the Chrony charm
+2. Enable NTS for the Chrony charm
+3. Use the Chrony charm as a time server
+
+## Prerequisites
+* LXD installed and bootstrapped.
+* A host machine with Juju version 3.4 or above and a Juju controller bootstrapped.
+
+## Create a Juju model
+Since the clock is a kernel function, we will need to use LXD virtual machines
+instead of containers.
+```
+juju add-model test-chrony
+juju set-model-constraints virt-type=virtual-machine
+```
+
+## Deploy the chrony charm and confirm the chrony charm are deployed
+1. Deploy the latest chrony charm.
+```
+juju deploy chrony --channel latest/edge
+```
+2. Run `juju status`. You should see something like this (IPs will be different):
+```
+Model          Controller  Cloud/Region      Version  SLA          Timestamp
+test-chrony    lxd         lxd/default       3.5.2    unsupported  07:20:46Z
+
+App     Version  Status   Scale  Charm   Channel      Rev  Exposed  Message
+chrony           blocked      1  chrony  latest/edge   35  no       no time source configured
+
+Unit       Workload  Agent  Machine  Public address  Ports    Message
+chrony/0*  blocked   idle   0        10.81.171.230   123/udp  no time source configured
+
+Machine  State    Address        Inst id        Base          AZ  Message
+0        started  10.81.171.230  juju-728b65-0  ubuntu@24.04      Running
+```
+
+## Configure the time source for chrony charm
+Since Chrony needs a time source before it can serve time to others, 
+we must configure a time source for the Chrony charm. 
+In this example, we will use an upstream NTP server (ntp.ubuntu.com) 
+as the time source and set the iburst option to true for this source.
+
+```
+juju config chrony "sources=ntp://ntp.ubuntu.com?iburst=true"
+```
+
+Once the source is configured, the chrony charm should enter the active state.
```

Comment:
Nitpick: Be more explicit that the charm shows as active after running `juju status`, so something like

> Once the source is configured, the chrony charm should enter the active state in the `juju status` table.

Reply:
Updated, thanks!